### ISSUE TITLE
Server Error When We Deploy Our Application to Google Cloud Platform

### ISSUE THREAD

**Issue by dailinshen** [dailinshen/coms4156_jumpstart#5]
We tried to deploy our application on Google Cloud Platform. Unfortunately, we followed the step in the project WIKI page. We successfully installed all the required package via requirements, initialized Google Credentials, and deployed the app. However, when we run “gcloud app browse”, our browser showed us an error message saying “Server Error. There is an error with server, and we cannot process your request. Please wait for 30 seconds and have another try.”

We googled for solutions and attempted several but so far, we haven’t fixed problem yet…..

**Comment by dailinshen** [comment]
Before we transplanted the completed Imhere application to Google Cloud Platform, all the unit test were executed without any failures or errors on our local machine. We also manually use our application locally, the UI looked great, the flow worked fine, the performance of all the new features were expected.

**Comment by dailinshen** [comment]
We debugged our application on Google Cloud Platform via Stackdriver. It showed us an error message saying somehting like this, 

`No module named ssl`

So we googled for solution. It seems like we need to specify we are using `ssl` library if our application is up there on the cloud. Hence, we added some lines to `app.yaml` file. It then looks like this,

```
runtime: python27
api_version: 1
threadsafe: true

handlers:
- url: /.*
  script: imhere.app

libraries:
- name: ssl
  version: 2.7
```

Then we deployed it to our google cloud. Tada, the issue fixed.
